### PR TITLE
Update onboarding flow

### PR DESCRIPTION
**Pull request by TobiasDeBruijn**

### CODE REVIEW

**Review by SilasPeters — APPROVED** [review]
De dingen die je door hebt gekruis staan idd in de PR

File: config/locales/mailings.nl.yml [hunk 1]
```diff
@@ -14,22 +14,22 @@ nl:
...
             description: Naast alle bovenstaande studie- en carrièregerelateerde activiteiten, organiseren we ook veel simpelweg gezellige activiteiten. Daarnaast is er wekelijks een gratis borrel!
             name: Gezelligheid
         reception_justification: Je ontvangt deze mail omdat je je hebt aangemeld voor onze machtig mooie studievereniging! Aan het eind van deze mail vind je de knop om je account in ons ledenportaal te activeren. Je kunt ook dit prachtige introductiepraatje overslaan en meteen naar beneden scrollen, dat zien we toch niet (of toch wel?).
         see_you_soon: Tot snel!
         welcome: Welkom bij Sticky!
-        wrap_up: Nieuwsgierig naar welke activiteiten we binnenkort organiseren? Meer informatie vind je in ons ledenportaal en je kunt je daar ook meteen inschrijven! Daarnaast kun je in dit ledenportaal ook je gegevens aanpassen en je tegoed voor snacks en drinken opwaarderen.
+        wrap_up: Nieuwsgierig naar welke activiteiten we binnenkort organiseren? Meer informatie vind je in ons ledenportaal %{koala_link_start} Koala %{link_end} en je kunt je daar ook meteen inschrijven! Daarnaast kun je in dit ledenportaal ook je gegevens aanpassen.
```
[the remark this answers is not in the document]
Comment (SpookyBoy99) [reply config/locales/mailings.nl.yml:32]: Technisch gezien moet er dan ook nog een comma achter :)

[the remark this answers is not in the document]
Comment (SilasPeters) [reply config/locales/mailings.nl.yml:32]: Ah moet verder lezen dan mijn neus lang is

**Review by SilasPeters — APPROVED** [review]

File: config/locales/mailings.en.yml [hunk 2]
```diff
@@ -14,22 +14,22 @@ en:
...
         activate_account: Activate account
-        activity_updates_html: 'Do you always want to stay informed about these activities? Join the %{facebook_group_link_start} Sticky-members Facebook group %{link_end} and like  %{facebook_page_link_start} our page %{link_end}! Aside from that, you can find all information that you could ever want on our website: %{sticky_site_link_start} svsticky.nl %{link_end}. You can also send a WhatsApp message to the Sticky Phone (+31617695025) saying "Hi + [your name], English" after saving it in your contacts. Then you’ll automatically be updated about future events!'
+        activity_updates_html: 'Do you always want to stay informed about these activities? Follow our %{instagram_page_link_start} Sticky-members Instagram %{link_end} and follow our %{linkedin_page_link_start} our page %{link_end}! Aside from that, you can find all information that you could ever want on our website: %{sticky_site_link_start} svsticky.nl %{link_end}. You can also join our %{whatsapp_promo_link_start} WhatsApp promotion channel %{link_end} to be notified about upcoming activities!'
```
Comment (SpookyBoy99) [inline config/locales/mailings.en.yml:17]: Shouldn't this be something like `Follow our %{instagram_page_link_start} Instagram %{link_end} and %{linkedin_page_link_start} LinkedIn %{link_end} for updates about Sticky!`?

**Review by SpookyBoy99 — COMMENTED** [review]
It still has some issues IMHO, but mostly stylistic. The hardcoded CSS will actually cause issues for people using light mode in their browsers though.

**Pull request merged**